.changelog/_unreleased.toml

```
@@ -45,3 +45,9 @@ id = "dddb0086-94f2-4b18-b73d-9816b2fc296f"
 type = "improvement"
 description = "[uv] Inject index environment variables to subprocesses"
 author = "antoine.broyelle@helsing.ai"
+
+[[entries]]
+id = "6a2d1204-cc1d-43f1-9b45-577aec55a104"
+type = "improvement"
+description = "Directly call `uv build` even when no \\[build-system\\] is specified"
```

Was this some escaping to pass as a command line argument to slap?
```suggestion
description = "Directly call `uv build` even when no [build-system] is specified"
```